### ISSUE TITLE
Reducing size of training set will make output pure gray picture.

### ISSUE THREAD

**Issue by CreepZzy** [zhixuhao/unet#74]
When I used my dataset to train the u-net, I found a weird phenomenon. When I used 16 images to train, the predict result will show a image like mask, a black and white image. However, when I used 14 images to train the model, the predict results became pure gray images. Did anyone face the same problem or have some solutions to this problem?

**Comment by helwilliams** [comment]
yes I have the same problem, I have 26 input ultrasound images and I get a low contrast image warning, if anyones solved this problem any information would be most helpful. Thanks

**Comment by CreepZzy** [comment]
> yes I have the same problem, I have 26 input ultrasound images and I get a low contrast image warning, if anyones solved this problem any information would be most helpful. Thanks

I found that the problem may resulted from randomness. Reduce the epochs and try to do the training multiple times. Sometimes it gave pure gray results, somtimes it can produce meaningful results when the loss function decrease significantly in the first epoch... So maybe you can run the code more times.

**Comment by jizhang02** [comment]
hello,
Why the input data size is different from the output size?
